docs/pages/versions/unversioned/sdk/in-app-purchases.md

```diff
@@ -213,17 +213,19 @@ if (!purchase.acknowledged) {
 }
 ```
 
-### `InAppPurchases.getPurchaseHistoryAsync(refresh?: boolean)`
+### `InAppPurchases.getPurchaseHistoryAsync(options)`
 
-Retrieves the user's previous purchase history.
+Retrieves the user's purchase history.
 
-On Android, if refresh is set to `true` it will make a network request and return up to one entry per item even if that purchase is expired, canceled, or consumed. Use this if you want to sync purchases across devices or see purchases that are expired or consumed. If refresh is `false` it relies on the Play Store cache. An important caveat is that the return type when refresh is `true` is actually a subset of when it's `false`. This is because Android returns a `PurchaseHistoryRecord` which only contains the purchase time, purchase token, and product ID, rather than all of the attributes found in the `InAppPurchase` type.
-
-On iOS, the refresh boolean is ignored. An important thing to note is that on iOS, Storekit actually creates a new transaction object every time you restore completed transactions, therefore the `purchaseTime` and `orderId` may be inaccurate if it's a restored purchase. If you need the original transaction's information you can use `originalPurchaseTime` and `originalOrderId`, but those will be 0 and an empty string respectively if it is the original transaction.
+Please note that on iOS, Storekit actually creates a new transaction object every time you restore completed transactions, therefore the `purchaseTime` and `orderId` may be inaccurate if it's a restored purchase. If you need the original transaction's information you can use `originalPurchaseTime` and `originalOrderId`, but those will be 0 and an empty string respectively if it is the original transaction.
 
```

Comment:
```suggestion

You should not call this method on launch because restoring purchases on iOS prompts for the user’s App Store credentials, which could interrupt the flow of your app.
```

Adding this into this PR to address https://github.com/expo/expo/issues/11294